README.rst

```diff
@@ -26,6 +26,8 @@ Documentation is available at `cirq.readthedocs.io <https://cirq.readthedocs.io>
 - `Documentation <https://cirq.readthedocs.io>`_
 - `Tutorial <https://cirq.readthedocs.io/en/stable/tutorial.html>`_
 
+Note: The (*default*) documentation for ``stable`` might be behind ``latest`` (see `here <https://cirq.readthedocs.io/en/latest/tutorial.html>`_), which can cause confusion if you are working on branches such as ``cirq-dev`` or ``cirq-release``.
```

Comment:
Mention what `stable` is (the docs that we just linked above).

Reply:
People who read this for the first time won't know what `cirq-dev` and `cirq-release` are.

Reply:
What should I say about stable? I looked at the docs and didn't see any explicit explanation on how you folks are releasing stable.

Something like this?

Note: ``stable`` is ???. Therefore, the documentation for ``stable`` 
might be behind ``latest`` (see `here <https://cirq.readthedocs.io/en/latest/tutorial.html>`_).

Reply:
What do you think about moving this sentence up? 

Documentation is available at cirq.readthedocs.io.  The (*default*) documentation for [stable](https://cirq.readthedocs.io/en/stable/) might be behind [latest](https://cirq.readthedocs.io/en/latest/tutorial.html).

Reply:
Done!